windows/src/robocorp/windows/_control_element.py

```diff
@@ -30,6 +30,22 @@
 DEFAULT_SEND_KEYS_INTERVAL = 0.01
 
 
+class _SentinelValidator:
+    """
+    Used to determine the validator function for ControlElement.set_value method.
+    If a validator is not passed then set_value_validator will be used as default.
+    """
```

Comment:
```suggestion
    """Used to determine the validator function for ControlElement.set_value method.
    
    If a validator is not passed then set_value_validator will be used as default.
    """
```
This is how it should be to match our latest docstring rules.

But why having a docstring here when it should have been a comment? (since `_SentinelValidator` belongs to the private API, which can be documented through comments, not docstrings)